themes.gitconfig

```diff
@@ -71,7 +71,7 @@
     minus-style = "#ff0080"
     plus-emph-style = "#008a81" bold "#00ffbf"
     plus-style = syntax "#cffff3"
-    syntax-theme = Github
+    syntax-theme = GitHub
```

Comment:
I'm not entirely certain that this change works, but I hope that the syntax-theme is case-insensitive